specification/cognitiveservices/Language.Conversations/common.tsp

```diff
@@ -76,7 +59,7 @@ enum StringIndexType {
 @doc("Represents List of Question Answers.")
 model AnswersResult {
     @doc("Represents Answer Result list.")
-    answers: Array<KnowledgeBaseAnswer>;
+    answers?: Array<KnowledgeBaseAnswer>;
```

Comment:
All those properties on `KnowledgeBaseAnswer` likely need this same treatment. Compare against the existing swagger to see if any are required.